store/storetest/user_store.go

```diff
@@ -3377,8 +3377,8 @@ func testUserStoreGetProfilesNotInTeam(t *testing.T, ss store.Store) {
 	// Ensure update at timestamp changes
 	time.Sleep(time.Millisecond * 10)
 
-	store.Must(ss.Team().RemoveMember(teamId, u1.Id))
-	store.Must(ss.Team().RemoveMember(teamId, u2.Id))
+	// store.Must(ss.Team().RemoveMember(teamId, u1.Id))
```

Comment:
You are not removing the users as before. We normally replace the `store.Must` with a call getting at least the error and checking the error with `require.Nil(t, err)`.

Reply:
Thank you @jespino , I had commented it out in the meantime as I tried to figure out a suitable replacement for that piece of code. I am going to update the code using your suggestion.